client/components/document.vue

```diff
@@ -72,6 +75,10 @@
     },
 
     methods: {
+      reloadEditor(key) {
```

Comment:
So this is just temporary?

Reply:
I added this just for reloading the `editor` component after the *Undo Changes* button is pressed. We need to reload the prosemirror editor state (basically rerun what's inside of `mounted`).